### DISCUSSION TITLE
TextureAtlasSprite comes out looking strange

### DISCUSSION THREAD

**Discussion by biskwikman** [bevyengine/bevy#4549] [Q&A]
Hello, I have a spritesheet of 32x64 sprites. I'm trying to display the first one, and it's almost working, but the bottom row of pixels seems to be stretched out to the bottom of the screen. This seems like a different issue than the sprite bleed issue, so I think it may be that my code is off. I'll post the relevant code and the result.

```rust
struct CharSheet(Handle<TextureAtlas>);

fn load_textures(
    mut commands: Commands,
    assets: Res<AssetServer>,
    mut texture_atlases: ResMut<Assets<TextureAtlas>>,
) {
    let image = assets.load("textures/main-char-sheet.png");
    let atlas = TextureAtlas::from_grid(
        image, 
        Vec2::new(32.0, 64.0), 
        10, 
        1, 
    );

    let atlas_handle = texture_atlases.add(atlas);

    commands.insert_resource(CharSheet(atlas_handle));
}

fn spawn_player(mut commands: Commands, character: Res<CharSheet>){
    let mut sprite = TextureAtlasSprite::new(1);
    sprite.custom_size = Some(Vec2::new(1.0, 2.0));

    commands.spawn_bundle(SpriteSheetBundle {
        sprite: sprite,
        texture_atlas: character.0.clone(),
        transform: Transform{
            translation: Vec3::new(0.0, 0.0, 900.0),
            ..Default::default()
        },
        ..Default::default()
    }).insert(Name::new("Player"));
}
```

![Screenshot from 2022-04-21 16-12-05](https://user-images.githubusercontent.com/17582234/164395799-0b7de740-4b6d-4821-a9f7-50b8d8ebad50.jpg)

as you can see, his 1px high feet have become very very long. I think this may have something to do with the `custom_size` method?

**Comment by The-DevBlog** [comment]
Whats the purpose of having 900.0 on the Z axis?
